app/src/renderer/js/preferences.js

```diff
@@ -94,6 +94,50 @@ document.addEventListener('DOMContentLoaded', () => {
     $j('#plugins-installed').html(html);
   }
 
+  function loadDefaultShareService() {
+    const shareServices = plugins.getShareServices();
+    const template = `
+      <select>
+        <option>Show editor [default]</option>
+        <% _.forEach(options, option => { %>
+            <option <%- option.selected ? 'selected' : '' %> value='<%- option.value %>'>
+              <%- option.title %> [<%- option.format %>]
+            </option>
+        <% }); %>
+      </select>
+    `;
+
+    const compiled = _.template(template);
+    const current = app.kap.settings.get('defaultShare');
+
+    const options = shareServices.reduce((all, service) => {
```

Comment:
rewriting it with `for..of` will make the code mutate, I prefer to write immutable code, but sure